perun-dispatcher/src/main/java/cz/metacentrum/perun/dispatcher/scheduling/TaskScheduler.java

```diff
@@ -214,6 +218,22 @@ private TaskSchedule getWaitingTaskSchedule() throws InterruptedException {
 		return taskSchedule;
 	}
 
+	/**
+	 * Method waiting for propagation of tasks to the engine to be resumed.
+	 * Called when propagation was suspended.
+	 */
+	private void waitForResumingPropagation() {
+		int sleepTime = 10000;
+		while (tasksManagerBl.isSuspendedTasksPropagation()) {
+			try {
```

Comment:
If Tasks propagation is suspended, we idle here in a cycle. Hence we won't get some debug messages about the pool and queue. I would like to have them logged in every cycle as waiting for TaskSchedule does.
```java
log.debug(schedulingPool.getReport());
log.debug("WaitingTasksQueue has {} normal Tasks and {} forced Tasks.", waitingTasksQueue.size(), waitingForcedTasksQueue.size());
```
Also, following log message is logged only once (before the cycle), so it might not be visible from the log, that propagation is currently suspended. Please repeat this message also in each cycle.
```java 
log.debug("Propagation of tasks is suspended.");
```